src/starcheck.pl

```diff
@@ -136,6 +138,23 @@
 my $dither_file= get_file("$par{dir}/History/DITHER.txt*",'dither'); 
 my $radmon_file= get_file("$par{dir}/History/RADMON.txt*", 'radmon');
 
+# Check for characteristics.  Ignore the get_file required vs not API and just pre-check
+# to see if there is characteristics
+my $char_file;
+my $char_glob = "$par{dir}/${sosa_dir_slash}mps/ode/characteristics/CHARACTERIS*";
```

Comment:
It is certaintly sufficient to check the root version since this file doesn't change for `vehicle`.  I read Brent's message as "we're going to do this unless anyone objects", so I expect that the current code will fail for NOV1615.